### ISSUE TITLE
Add to Structure

### ISSUE THREAD

**Issue by wispoz** [dabeng/OrgChart#44]
Hello, its hard to add something like that?
![strcc](https://cloud.githubusercontent.com/assets/156639/17431032/6dead49e-5b00-11e6-93fe-98cd2a99bd4f.png)

name: Kobayashi 

simple if its org chart and one employee is assistant to another.

**Comment by dabeng** [comment]
Hi. It's not a big deal. You can use the following code snippets to implement this feature:
```js
    $('#chart-container').orgchart({
      'data' : datascource,
      'createNode': function($node, data) {
        if (data.assistant) {
          $node.append(<div class="assistant">Kobayashi</div>);
         }
      }
    });

```

**Comment by wispoz** [comment]
Ye, but how it will work with rendered structure :
we have space between node1 and node2 we must have space to add assistant

**Comment by dabeng** [comment]
It doesn't matter. You just need to append some styles of node to enlarge the distance between normal nodes in horizontally and then you will got the large enough space to place the assistant node. For example:
```css
.orgchart .node {
  margin: 0 30px;
}
```
On the other hand, you also need to append relative position settings for assistant node.

**Comment by dabeng** [comment]
Another proposal, I think it will be better if we remain the distance between normal nodes and add show/hide effects for assistant nodes. If I were you, I would add icon standing for assistant role into the content section of normal node and toggle the assistant node by clicking the icon. By doing so, we can save lots of space to show more nodes and hierarchies.

**Comment by HariBabuThubati** [comment]
drag and drop in IE is not working.can you do needful.

**Comment by dabeng** [comment]
Hi @HariBabuThubati , which version of your IE browser ?

**Comment by HariBabuThubati** [comment]
Hi dabeng, IE11 i am using.

**Comment by dabeng** [comment]
in fact, I have tested drag&drop feature on IE 11, it works well. So I wanna know did you change any core code snippets of drag&drop example?

**Comment by HariBabuThubati** [comment]
Hi dabeng, i just download and ran the example for drag drop.

**Comment by fahad4fbi** [comment]
Hi @wispoz ,
This line is lib. is responsible for drag and drop problem in IE.
`event.originalEvent.dataTransfer.setData('text/html', 'hack for firefox');` // line 647

Kindly wrap this line inside a check, so it only executes in firefox. You will be good to go.

**Comment by dabeng** [comment]
Thank you guys for your deep insight. This bug has been fixed in version 1.1.4.

**Comment by naylee** [comment]
Hi @dabeng, i want to have chart similar as what @wispoz did. I already tried your suggestions too. 
Here is my orgchart:
![image](https://user-images.githubusercontent.com/35753076/39804040-3135c3c8-539d-11e8-8492-788acaea9818.png)

But, since Employee 2 only need 1 assistant-node, so I use margin-right only and here is the result:

![image](https://user-images.githubusercontent.com/35753076/39804135-68652186-539d-11e8-802e-501805a7902e.png)

The problem is, I want the 'top-line' in the middle of 'Employee 2' node not in between 'Employee 2' node and asisstant-node.

Is it possible to do that?
